app/models/openscap_result.rb

```diff
@@ -0,0 +1,68 @@
+require 'openscap'
+require 'openscap/ds/arf'
+require 'openscap/xccdf/benchmark'
```

Comment:
@jrafanie do you mean the binary it uses(oscap)? could this be working only because I have the binary installed?
If that is the case a dependency would have to be added on openscap-scanner. @isimluk is that correct?